freedommaker/builder.py

```diff
@@ -125,6 +132,25 @@ def _get_image_base_name(self):
                 build_stamp=self.arguments.build_stamp, machine=self.machine,
                 architecture=self.architecture)
 
+    def get_temp_image_file(self):
+        """Get the temporary path to where the image should be built.
+
+        If building to RAM is enabled, create a temporary directory, mount
+        tmpfs in it and return a path in that directory. This is so that builds
+        that happen in RAM will be faster.
+
+        If building to RAM is disabled, append .temp to the final file name and
+        return it.
+
+        """
+        if not self.arguments.build_in_ram:
+            return self.image_file + '.temp'
+
+        self.ram_directory = tempfile.TemporaryDirectory()
+        self._run(['mount', '-t', 'tmpfs', 'tmpfs', self.ram_directory.name])
```

Comment:
When running freedommaker as normal user, I get a build error:
```
subprocess.CalledProcessError: Command '['mount', '-t', 'tmpfs', 'tmpfs', '/tmp/tmpylh69806']' returned non-zero exit status 1
```
Running this command directly shows the reason as:
```
mount: only root can use "--types" option
```
Could "sudo" be added here?